packages/@dcl/sdk-commands/src/logic/bundle.ts

```diff
@@ -300,23 +300,32 @@ function runTypeChecker(components: BundleComponents, options: CompileOptions) {
   })
   const typeCheckerFuture = future<number>()
 
+  let stdOutput = ''
+  ts.stdout?.on('data', (data: string) => {
+    stdOutput += data
+  })
```

Comment:
`stdOutput` will grow indefinitely, causing a memory leak since it will never be garbage collected, not even when there is a ts error. 
If we really want this, one option is to make `stdOutput` to be [tailed](https://man7.org/linux/man-pages/man1/tail.1.html) to a hardcoded number of characters